apps/etterlatte-api/.nais/dev.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: "nais.io/v1alpha1"
 kind: "Application"
 metadata:
-  name: etterlatte-samordning-vedtak
+  name: etterlatte-api
```

Comment:
Kan man ha to apper med samme navn i samme miljø?

Reply:
Oisann, har visst vært litt kjapp med find-and-replace. Skal korrigere!